docs/application/design/tv/design-principles.md

```diff
@@ -7,6 +7,76 @@ elements help the user get information easily and enjoy content fully.
 
 There are 3 design principles used in Tizen 4.0 TVs:
 
--   [Simplicity](design-principles/simplicity.md)
--   [Natural    intuitiveness](design-principles/natural-intuitiveness.md)
--   [Breathing space](design-principles/breathing-space.md)
+-   [Simplicity](#simplicity)
+-   [Natural intuitiveness](#natural-intuitiveness)
+-   [Breathing space](#breathing-space)
+
+## Simplicity
+
+Using many decorative elements on the screen at the same time can confuse and distract the user.
+
+Simplify your application design to allow the user to clearly identify the content.
+
+### Circular Icons
+
+Tizen 4.0 TV icons are circular, covering a smaller area than
+rectangular icons. Consequently, the user can see more of the screen
+background.
+
+![circlur icons](media/dp_01_sim_circle_re-850x478.png)
+
+*Circular icons taking less space on the screen.*
+
+
+### Hidden Inactive Icons
+
+An inactive icon partly hides itself to take up even less space on the screen. This approach also allows the user to recognize active icons without visual interruption from inactive icons.
+
+![hidden inactive icons](media/dp_02_sim_hide.png)
+
+*Inactive icons are hidden.*
+
+
+### No Unnecessary Decorations
+
+Within functional design elements, clean up unnecessary decorations, such as a divider bar. Instead, use more functionally-effective visual expressions or separate information. It makes the UI look cleaner and more organized.
+
+![No divider bar](media/dp_03_sim_clean-850x478.png)
+
+*No divider bar between different pieces of information.*
+
+## Natural Intuitiveness
+
+Enhance the usability and make the interface naturally familiar to the user by, for example, using [shadow effects](natural-intuitiveness/shadow-effect.md). This allows the user to access content and information easily with a pleasing natural visual balance.
```

Comment:
It looks like this `[shadow effects](natural-intuitiveness/shadow-effect.md)` is leading to the section right below. Dont' we need to remove this link?

Reply:
done